packages/decentraland-ecs/src/setupProxy.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path')
+const fs = require('fs')
+
+module.exports = function (dcl, app, express) {
+    const dclKernelPath = path.resolve(dcl.getWorkingDir(), 'node_modules', 'decentraland-kernel');
```

Comment:
Use `require.resolve` to recursively find the path using node_modules resolutor embedded in Node.js
```suggestion
    const dclKernelPath = require.resolve('decentraland-kernel', {paths: [dcl.getWorkingDir()]})
```

Reply:
That doesn't resolve the directory instead it resolves the module file if exists (not the decentraland-kernel case). Maybe, `const dclKernelPath = path.dirname(require.resolve('decentraland-kernel/package.json', {paths: [dcl.getWorkingDir()]}))` ?

Reply:
RIGHT, your suggestion is correct :shipit: